shotover-proxy/src/transforms/distributed/consistent_scatter.rs

```diff
@@ -52,49 +51,41 @@ impl ConsistentScatterConfig {
     }
 }
 
-fn get_timestamp(frag: &QueryResponse) -> i64 {
-    debug!("\n\n {:#?} \n\n", frag.response_meta);
-    if let Some(MessageValue::Document(meta)) = frag.response_meta.as_ref() {
-        if let Some(t) = meta.get("timestamp") {
-            if let MessageValue::Integer(i, _) = t {
-                return *i;
-            }
-            return 0;
-        }
+/// Get the size of the frame within a `Message`
+fn get_size(message: &mut Message) -> usize {
+    // TODO is this the right approach? Need to implement a similar method on CassandraFrame
```

Comment:
I think get_size is a bad design, so no point trying to properly implement it for cassandra.